### DISCUSSION TITLE
cmdline completion does not appear below the popup.

### DISCUSSION THREAD

**Discussion by izkluxcvy** [folke/noice.nvim#242] [Q&A]
Hello.
Thank you for the great plugin.
I am using nvim-cmp cmdline completion.
I have installed the latest version of noice.nvim, but the completion appears where it should.
Disabling smart_move did not fix it.
The config is set to (https://github.com/folke/noice.nvim/wiki/Configuration-Recipes#display-the-cmdline-and-popupmenu-together).
The version of neovim is v0.8.0-dev.
What am I missing?
Sorry if I missed reading the documentation.

![image](https://user-images.githubusercontent.com/67828134/204083161-8d54e817-b9c3-46dc-828f-9f26f8c71189.png)

**Comment by TaterDoge** [comment]
How is this problem solved?
